blocks/inspector-controls/radio-control/index.js

```diff
@@ -32,7 +32,7 @@ function RadioControl( { label, selected, help, instanceId, onChange, options =
 						name={ id }
 						value={ option.value }
 						onChange={ onChangeValue }
-						selected={ option.value === selected }
+						defaultChecked={ option.value === selected }
```

Comment:
Why do we need this change? we're using "value" to make the radio control "controllable". Same for the select one.